aspects/aspects.go

```diff
@@ -312,7 +294,7 @@ func (a *Aspect) Set(databag DataBag, request string, value interface{}) error {
 		}
 
 		if !accessPatt.isWriteable() {
-			return &InvalidAccessError{RequestedAccess: write, FieldAccess: accessPatt.access, Field: request}
+			continue
```

Comment:
Instead of just continuing here, would it be good to additionally set some sort of flag that will tell you what kind of error to return when `len(paths) == 0`? That way, you could say something like `rules that match request do not have write access`.

Maybe we don't want to expose that information though, unsure.

Reply:
That was my initial intention but I ended up just simplifying it because it was still a request/rules mismatch and we didn't have specific errors for other types of mismatch. I guess it might be an easy thing to overlook, so it would be more user friendly to have a specific error message for it (which is reason enough to do it)